### ISSUE TITLE
New ontology Alzheimer's Disease Ontology

### ISSUE THREAD

**Issue by BideZ** [OBOFoundry/OBOFoundry.github.io#1806]
<a href="https://obofoundry.org/obo-nor.github.io/dashboard/ado/dashboard.html"><img src="https://img.shields.io/endpoint?url=https%3A%2F%2Fraw.githubusercontent.com%2FOBOFoundry%2Fobo-nor.github.io%2Fmaster%2Fdashboard%2Fado%2Fdashboard-qc-badge.json" /></a>

## Ontology title
Alzheimer's Disease Ontology

## Requested ID space
ADO

## Ontology location
https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO/tree/main/Releases/2022-06-11/ADO.owl

## Contact person
Name: Alpha Tom Kodamullil
Email address: alpha.tom.kodamullil@scai.fraunhofer.de
GitHub username: akodamullil

## Issue tracker
https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO/issues
## Version Controlled Repository
https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO
## Ontology license

- [ ] CC0 (public domain)
- [X] CC-BY (version 3 or later)
- [ ] Other: *please specify*

## Available ontology formats
.owl

## What domain is the ontology intended to cover?
Alzheimer's Disease

## Related OBO Foundry ontologies
SYMP, DOID, UBERON 

## Intended use/related projects
A disease ontology to classify Alzheimer's Disease and calculate semantic similarities in the domain of Alzheimer's Disease, Annotation of medical texts.

## Data source
Several sources from domain experts assembled by Fraunhofer SCAI.

## Additional comments or remarks
Created by the team for applied semantics at Fraunhofer SCAI
## OBO Foundry pre-registration checklist

To be considered for inclusion in the OBO Foundry, an ontology must meet certain requirements, as described in in [the registration process instructions](http://obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html) and the [registration review checklist](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/RegistrationChecklist.md). To ensure you are aware of some of its key points, please review the checklist below.

You can either check a box by submitted the request first and then using the GitHub interface, or replacing the `- [ ]` by `- [X]` in the following.

- [X] I have read and understood [the registration process instructions](http://obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html) and the [registration checklist](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/RegistrationChecklist.md) 
- [X] There is no other ontology in the OBO Foundry which would be an appropriate place for my terms. If there was, I have contacted the editors, and we decided in mutual agreement that a separate ontology is more appropriate.
- [X] My ontology has a specific release file with a version IRI and a dc:license annotation, serialised in RDF/XML.
- [X] I understand that term definitions, while not mandatory, are key to understanding the intentions of a term especially when the ontology is used in curation. I made sure that a reasonable majority of terms in my ontology have definitions, using the [IAO:0000115](http://www.ontobee.org/ontology/IAO?iri=http://purl.obolibrary.org/obo/IAO_0000115) property.
- [X] For every term in my ontology, I checked whether another OBO Foundry ontology has one with the same meaning. If so, I re-used that term directly (not by cross-reference, by directly using the IRI).
- [X] For all relationship properties (Object and Data Property) I checked whether [RO](http://www.ontobee.org/ontology/catalog/RO?iri=http://www.w3.org/2002/07/owl%23ObjectProperty) includes an appropriate one. I understand that aligning with RO is an essential part of the overall alignment between OBO ontologies!
- [X] For the selection of appropriate annotation properties, I looked at [OMO](http://www.ontobee.org/ontology/catalog/OMO?iri=http://www.w3.org/2002/07/owl%23AnnotationProperty) first. I understand that aligning ontology metadata and term-level metadata is essential for cross-integration of OBO ontologies. 
- [X] If I was not sure about the meaning of any of the checkboxes above, I have consulted with a member of the OBO Foundry for advice.
- [X] The requested ID space does not conflict with another ID space found in other registries such as BioPortal and the Bioregistry, see [here](https://obofoundry.org/id-policy) for a complete list.

## Metadata

Please fill in the following metadata record which will be used by the OBO Foundry website. Note that the values shown are just examples, for example `yourfourletterid` could be something like `aism`, `cohm`, `mondo` (it does not have to be four letters). `your_domain_like_for_example_anatomy` could be simply `anatomy`, and the license should be whatever your actual license is. An example can be found [here](https://github.com/OBOFoundry/OBOFoundry.github.io/edit/master/ontology/amphx.md), but you really only need to fill in the metadata mentioned here.

```
id: ADO
title: Alzheimer's Disease Ontology
contact: 
  email: alpha.tom.kodamullil@scai.fraunhofer.de
  label: Alpha Tom Kodamullil
description: Alzheimer's Disease Ontology is a knowledge-based ontology that encompasses varieties of concepts related to Alzheimer'S Disease, foundamentally structured by upper level Basic Formal Ontology(BFO). This Ontology is enriched by the interrelational entities that demonstrate the nextwork of the understanding on Alzheimer's disease and can be readily applied for text mining.
domain: Alzheimer's Disease 
homepage: https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO
products:
  - id: ADO.owl
dependencies:
  - id: BFO
tracker: https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO/issues
license:
  url: https://creativecommons.org/licenses/by/4.0/
  label: CC-BY
usages:
  - user: -
    description: -
```

**Comment by BideZ** [comment]
@matentzn Thank you for the feedback. It should be accessible now.

**Comment by matentzn** [comment]
@BideZ I am very happy you used all OBO ids, I hope you don't mind me sending you some feedback in your issue tracker:

- [ ] https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO/issues/1
- [ ] https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO/issues/2
- [ ] https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO/issues/3

**Comment by matentzn** [comment]
@BideZ thank you for your submission. thank you very much!

Before we do our content review, we would like to invite you to make sure that your ontology passes the minimum QC threshold (or provide reasons why you don't want to / cannot):

https://obofoundry.github.io/obo-nor.github.io/dashboard/index.html

You can find the reports specific to ADO here:
https://obofoundry.github.io/obo-nor.github.io/dashboard/ado/dashboard.html

- [ ] The version IRI is not in the required format

Let us know when that is done (of if you don't want to do it), and we will commence with the review.

- [ ] I note that you have a prefix clash with https://bioportal.bioontology.org/ontologies/ADO, but its seems that is your ontology. Please confirm that is so.

**Comment by matentzn** [comment]
Two questions I have based on a cursory look through your metadata:

- You say ADO is aligned with BFO, but it is only very partially aligned with it, see also https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO/issues/3
- "This Ontology is enriched by the interrelational entities that demonstrate the nextwork of the understanding" - what does that mean? You added relationships that were not in the ontologies? Can you give an example?

**Comment by BideZ** [comment]
@matentzn Thank you for your feedback. We are currently trying to resolve some issues regarding the dangling terms(detailed explanation found in ADO issue 3). 
I have looked through the dashboard report and here are some feedbacks: 
Versioning: We want to stick with the version identifier (surfix 1.0.0)we put there due to internal reasons, and I hope it is okay.
Plurality of Users: ADO is not yet utilized by other independent institutes/organizations which could be added in the future, though. Please let me know if it is fine to stay like this or we have to add usages before the reviewing process takes place.

**Comment by BideZ** [comment]
@matentzn Regarding the last comment:

- Only part of BFO is aligned with our ADO because we want to keep it as simple as possible and only the most relavent BFO terms were selected. 
- We have some object properties describing the relationships/interactions between terms in Alzheimer's Disease, such as 'molecularly interacts with' and 'is carrier of'. This is what we meant by "This Ontology is enriched by the interrelational entities that demonstrate the nextwork of the understanding"

**Comment by matentzn** [comment]
@BideZ We will assign a reviewer to ADO at the next OBO Foundry call, which should give you some time to address https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO/issues/1 (this will have to be addressed before we start with the review). Note: None of what I told you here so far has anything to with the review - these are just gatekeeping checks to decide whether to review an ontology in the first place.

The next OBO Foundry call is on the 22nd March, so you should hear from us at some point soon after that!

**Comment by BideZ** [comment]
@matentzn Thank you very much. We will leave a comment here as soon as the issues are resolved, or if there is any updates. Besides, the ADO you find in BioPortal was developed by our team. However, it was not constructed with OBO approved terms. We will discuss this with our team and see if we are changing the prefix, or we take this ADO as an update version.

**Comment by matentzn** [comment]
Yes thanks, that would be very important - it is against our principles to find one ADO here and another on BioPortal!

**Comment by BideZ** [comment]
@matentzn Regarding clashing prefix on Bioportal: the newly submitted ADO will be the update version of the ADO found on Bioportal. Thank you!

**Comment by BideZ** [comment]
@matentzn Here are some updates about the issues on ADO:

- Regarding prefix clash: As we discussed with our Team, the ADO that we submitted over two weeks ago should be an updated version of the one found on Bioportal. So now we changed the surfix to 2.0.0 so it could be indicated. 

- Regarding usages (QC dashboard): There are currently no organizations/institutes that are using this unpublished ADO, yet we could add some in the furture after its publication or so. 

- Dangling terms: We successfully removed the dangling terms/put them in the corresponding positions. The new ADO was pushed.

- Submission of ObjectProperty to RO: We are considering submitting the object properties to RO. Could you please specify how they should be submitted? Should I include the object properties only in an owl file and make an issue? 

Thank you very much!
I am looking forward to your feedbacks!

**Comment by matentzn** [comment]
@BideZ thank you for cleaning up, next step is for the OBO Foundry to assign a reviewer to your ontology. We will come back to you!

> We are considering submitting the object properties to RO. Could you please specify how they should be submitted?

You can fill in this table:
https://github.com/oborel/obo-relations/blob/master/src/templates/new_relations.tsv

And make a pull request on RO! It will generate the related axioms automatically.

Before you do, make a separate pull request here: https://github.com/oborel/obo-relations/blob/master/src/ontology/ro-idranges.owl and tag me in it! This is to give you and your team your own ID range, so you do not clash with others.

**Comment by BideZ** [comment]
@matentzn Thank you for the reply. I saw this line in the link you sent 'EquivalentTo: xsd:integer[...]' 
Is this the range of number of relations we submit?

**Comment by matentzn** [comment]
Yes, you just copy another persons range and give yourself a few ids in the equivalentTo statement. You change the range ID, add your name, and it's ok! I will fix anything should it be necessary :)

**Comment by BideZ** [comment]
@matentzn Thank you! We already created a pull request in this regard. The relations to be submitted include the ones from both ADO and Epilepsy Ontology (EPIO). Should we wait for the assignment of relation IDs or can we already start filling in the table you mentioned earlier?

**Comment by cstoeckert** [comment]
I have posted issues at ADO found while reviewing the ontology. See https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO/issues/6
https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO/issues/5

**Comment by cstoeckert** [comment]
summary: The ADO is intended to be used for tagging Alzheimer’s Disease literature for NLP. The ontology does not include new classes just imported ones that might be useful for their application. The value added appears to be ~10 object properties which have been submitted to RO. Problems were found in their imports (e.g., placed PATO colors under race) and in the object properties and reported as issues on their tracker. 
@BideZ has responded to the issues raised and has made some improvements. However, despite back and forth discussions (see https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO/issues/6) and changes there are still fundamental problems with the object properties that are essentially the value added of the ontology. An example is isClassifiedSubtypeFor defined as "The subject is classified as a subtype of the object." It is used for: 'Alzheimer disease 16' isClassifiedSubtypeFor some 'Alzheimer's disease' ('Alzheimer disease 16' is SubClass Of "Alzheimer disease'). 
These same problems were also mentioned in the RO slack channel.
https://obo-communitygroup.slack.com/archives/C01P3D02U1H/p1647468947402249?thread_ts=1647467919.741139&cid=C01P3D02U1H

Because of this I can not recommend acceptance until the object property issues are better addressed.

**Comment by BideZ** [comment]
@cstoeckert Thank you for the feedback. After an internal discussion with our team and an overall evaluation, we have decided to remove 'isClassifiedSubclassOf' completely from ADO. One can get access to the latest version through https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO/blob/main/Releases/2.0.0/ADO.owl

**Comment by nlharris** [comment]
@cstoeckert does that decision address your concerns?

**Comment by cstoeckert** [comment]
The last issue about 'is carrier of' has also been addressed.

**Comment by cstoeckert** [comment]
@BideZ @akodamullil
Thank you again for your ontology submission to the OBO Foundry. We are happy to inform you that your ontology (ADO) has been accepted following discussion in the OBO Operations Committee meeting, 14-06-2022. Before including it into the OBO ontology registry you need to complete the following steps.
Create a metadata record for your ontology to be included in the registry
Create a new file in this directory (there is an “Add file” button in the top right): https://github.com/OBOFoundry/OBOFoundry.github.io/tree/master/ontology, called ADO.md
Obtain the already curated metadata that relates to your ontology from https://github.com/OBOFoundry/obo-nor.github.io/blob/master/dashboard-config.yml
Create a pull request to add the metadata record. This pull request should include a link to this issue (the New Ontology Request issue).
Here is an example record for the PATO ontology: https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/ontology/pato.md?plain=1
Your metadata will be reviewed and merged by a member of the OBO Foundry Operations Committee
Create a PURL registry entry for your ontology.
Go https://github.com/OBOFoundry/purl.obolibrary.org/tree/master/config, click “Add file” and add a file named ADO.yml.
Add the desired configuration.
Make a pull request with a link to this issue
See here for an example: https://github.com/OBOFoundry/purl.obolibrary.org/blob/master/config/pato.yml

Please note that this acceptance is based on the understanding that you will continue to address the issues raised including following up on the submission of properties to RO. I will also send this info to the email address of the contact person listed above. 
Thanks!

**Comment by nlharris** [comment]
What are the next steps for this ontology?

**Comment by matentzn** [comment]
Nothing: its all done: https://obofoundry.org/ontology/ado.html

**Comment by BideZ** [comment]
@matentzn Dear Mr. Matentzoglu,
Sorry to mention questions in a closed issue. I would like to ask if I need to create a new issue here for the updates for ADO ontology. We added a few new terms to the ADO ontology this year and would like the latest version to appear on OLS andb Ontobee. 

Besides, ADO doesn't seem to show properly on OLS :  https://www.ebi.ac.uk/ols/ontologies/ado. Do you have an idea why this has happened? Should I also report this as a new issue in Github? 
Best,
Bide

**Comment by matentzn** [comment]
This is the wrong tracker for this.. Its also an old link, this is the new one: https://www.ebi.ac.uk/ols4/ontologies/ado

These seem a few problems here, and I don't know how they came to be. Many ADO classes don have labels, like https://www.ebi.ac.uk/ols4/ontologies/ado/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FADO_0000032

This is where you configure the PURLs: https://github.com/OBOFoundry/purl.obolibrary.org/blob/master/config/ado.yml It is your responsibility but you can join OBO slack and ask for help there

**Comment by nlharris** [comment]
As Nico suggested, these issues have nothing to do with OBO Foundry. I hope you can find the right channels to get your issues fixed, BideZ!
